contents/careers/index.md

```diff
@@ -107,25 +107,13 @@ If you do *not* see a specific role listed, that really doesn't mean we won't ha
 
 We take this application seriously, and that's why it's listed above our other roles - you won't just end up in a candidate database, we will either hire you, reject you, or give you some insight into when may make more sense to apply.
 
-## Front-End Developer
+## Open Roles
 
-PostHog is building its first design team. We are looking for a proactive, fast, and passionate UX Developer to take our design to a world class level. Our product and website went viral with a basic MVP, and we're determined to be best-in-class with our UX.
+Our team is proactively looking for the following:
 
-The role would involve:
+* Front-End Developer
```

Comment:
Too late for this PR, but those could have been links. If you're viewing from a phone, it's not obvious that you find them in the careers submenu

Reply:
fixed by #547